docs/data_management/storage.md

```diff
@@ -59,10 +59,19 @@ Please provide the following information. Missing information can delay allocati
 - **Internal UAB Collaborator Organizations:** The name(s) of any other UAB organizations participating in the Shared Storage.
 - **External Collaborator Organizations:** The name(s) of any external organizations participating in the Shared Storage.
 - **Regulatory Requirements:** List any regulatory requirements or agencies affecting data to be stored in the space. Possibilities include, but are not limited to: IRB, EHR, HIPAA, PHI, FERPA.
-- **Name of Shared Storage:** Please give us a short, memorable name that is specific to your organization but general to your purpose.
+- **Name of Shared Storage:** Please give us a generic name but specific to your project/lab.
```

Comment:
Remove "but", so it reads "generic name that is specific to your project/lab"

Reply:
Changes incorporated